### PR TITLE
Added sceAVConfigSetDisplayColorSpaceMode to header

### PR DESCRIPTION
**Pull request by Electric1447**

### CODE REVIEW

File: include/psp2/avconfig.h [hunk 1]
```diff
@@ -65,6 +65,16 @@ int sceAVConfigSetSystemVol(int volume);
  *
  */
 int sceAVConfigMuteOn(void);
+  
+/**
+ * Sets the color space mode on runtime.
+ *
+ * @param colorSpaceMode - Color Space Mode that the screen will be set to (0 defualt, 1 high contrast mode).
```
Comment (Rinnegatamante) [inline include/psp2/avconfig.h:72]: Maybe would be cool to have an enum for those constants.

Comment (Electric1447) [reply include/psp2/avconfig.h]: Fixed typo.
How do I add the enum?

Comment (Rinnegatamante) [reply include/psp2/avconfig.h]: https://github.com/vitasdk/vita-headers/blob/master/include/psp2/audioout.h#L35-L42 Something like this

Comment (Electric1447) [reply include/psp2/avconfig.h]: done

File: include/psp2/avconfig.h [hunk 2]
```diff
@@ -13,6 +13,11 @@
 extern "C" {
 #endif
 
+typedef enum SceColorSpaceMode {
```
Comment (Rinnegatamante) [inline include/psp2/avconfig.h:16]: Add refering to the module (SceAvConfigColorSpaceMode for xample)

File: include/psp2/avconfig.h [hunk 3]
```diff
@@ -65,6 +70,16 @@ int sceAVConfigSetSystemVol(int volume);
...
  */
 int sceAVConfigMuteOn(void);
+  
+/**
+ * Sets the color space mode on runtime.
+ *
+ * @param[in] csm - see ::SceColorSpaceMode()
+ *
+ * @return 0 on success, < 0 on error.
+ * @note - This does not change color_space_mode in the registery.
```
Comment (Rinnegatamante) [inline include/psp2/avconfig.h:80]: Typo registery -> registry

Comment (Electric1447) [reply include/psp2/avconfig.h]: 👍

**Pull request merged**